open-vulnerability-clients/src/main/java/io/github/jeremylong/openvulnerability/client/nvd/Node.java

```diff
@@ -33,6 +34,7 @@
  * Defines a configuration node in an NVD applicability statement.
  */
 @JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
```suggestion
```

Reply:
Retained as the schema allows additional properties: https://github.com/jeremylong/Open-Vulnerability-Project/blob/83189a18225029b1cb535f747e6b7662acfd0ecd/open-vulnerability-clients/src/main/resources/json/cve_api_json_2.0.schema#L394-L418